cartography/intel/aws/ecr.py

```diff
@@ -0,0 +1,220 @@
+import logging
+
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def get_ecr_repositories(boto3_session, region):
```

Comment:
Would it be possible to add the `@aws_handle_regions` decorator please?

Reply:
Done